src/applications/vaos/tests/reducers/newAppointment.unit.spec.js

```diff
@@ -431,7 +431,7 @@ describe('VAOS reducer: newAppointment', () => {
       expect(newState.facilityDetailsStatus).to.equal(FETCH_STATUS.loading);
     });
 
-    it('should set facility detail loading', () => {
+    it('should update state with facility details set set facilityDetailsStatus to succeeded', () => {
```

Comment:
This looks like a typo:

```suggestion
    it('should update state with facility details and set facilityDetailsStatus to succeeded', () => {
```